src/main/java/com/yahoo/sketches/counting/FrequentItems.java

```diff
@@ -0,0 +1,107 @@
+package com.yahoo.sketches.counting;
+
+import java.util.ArrayList;
```

Comment:
Argh... I actually though 2M/sec was good :(
This never intended to be an optimized implementation though. I think I know how to implement some both faster and more general. That will come next though.